src/services/__tests__/helpersNextEcercise.spec.tsx

```diff
@@ -0,0 +1,105 @@
+import { loadDisciplines } from '../../hooks/useLoadDisciplines'
```

Comment:
File should be called `helpers.spec.ts` and another `describe` block should be used instead.

Reply:
Also I tried this, but the test in helpers.spec.tsx needs unmocked/real version of of AsyncStorage, wheres the tests for NextExercise (in this file) need a mocked version of it. As far as I know (and tired out) it is not possible to mock stuff only for some tests in a file.

Reply:
It is somehow possible (I think by using doMock and restoreMock or similar), but I agree, it is a real pain. However, the test that is currently in `helpers.spec.ts` in your branch was moved in #284 to the async storage test so you can move it without problems. Also the "real" async storage in tests should already be mocked afaik.

Reply:
Tried doMock, but couldn't get it working, but as you mentioned no need any more as it was moved.